### DISCUSSION TITLE
Have you submitted this yet?

### DISCUSSION THREAD

**Discussion by rljacobson** [Validark/DynSDT#4] [Q&A] [answered]
This is excellent. Have you submitted it anywhere yet? I couldn't find it on the [arXiv](https://arxiv.org). At the very least you should upload it there.

Also, dude, those animations are above and beyond. So cool!

**Comment by Validark** [comment, marked as answer]
Thank you so much for the compliments! I'll move this to the front of my to-do list for this week! 

Fun fact: I think the diagrams and animations are the most important part (aside from, I hope, encouraging people to think more creatively about tries). I actually got the idea for the final algorithm while working on the animation for the Completion Trie!

**Reply by Validark** [reply]
Alright, so I have had a few interviews this week that got scheduled after I posted my original reply, and I've prioritized those, but I have a break in the clouds on the horizon. I will get to this, I swear!
